internal/pkg/describe/static_site.go

```diff
@@ -0,0 +1,69 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"fmt"
+
+	"github.com/dustin/go-humanize/english"
+)
+
+const (
+	staticSiteOutputCFDomainName    = "CloudFrontDistributionDomainName"
+	staticSiteOutputCFAltDomainName = "CloudFrontDistributionAlternativeDomainName"
```

Comment:
Question! Are they supposed to be in the CloudFormation `Outputs` section?

Reply:
Yeah i think so because they are attributes of the dedicated CF just like we output the attributes of the LB in env stack.

Reply:
I guess this is not done yet? Adding `CloudFrontDistributionDomainName` and `CloudFrontDistributionAlternativeDomainName` into the `Outputs` section

Reply:
I think i added them https://github.com/aws/copilot-cli/pull/4870/files#diff-15ae3d1d11e7d786765fd0df1a1cdd256c6083500fc8c1e0f3856ffa7c4716f9R507-R517

Reply:
:homer-disappearing